example_experiments/python/genetic_algorithm.py

```diff
@@ -21,14 +21,14 @@
 Example settings for a run that demonstrates this:
 
 Info:
-File Output: geneticRes.csv
+Trial's Extra File: geneticRes.csv
```

Comment:
I thought `Trial's Extra File` was the replacement for `Result Output`?

Reply:
Yup should `Trial Result`, Was just copy and pasting